low-level/files/opt/seagate/sspl/bin/generate_sspl_bundle/sspl_bundle_generate

```diff
@@ -325,11 +329,28 @@ def sspl_bundle_cleanup():
     if os.path.exists(sspl_bundle_tmp):
         shutil.rmtree(sspl_bundle_tmp)
 
+def validate_sb_request():
+    """Validating upcomming sb request."""
+    valid_request = True
```

Comment:
Its not just the reboot which will interrupt. It can be crash or CTRL+C which is not related to the reboot.
Logic that should be used is... 

sspl_support_bundle() {
    if (exists /tmp/sspl_support_bundle) { 
         pid = read(/tmp/sspl_support_bundle)
         Check if process with pid exists. If so, exit as other process is running. 
    }
    Create /tmp/sspl_support_bundle and update current process pid
    ... Support Bundle....
}

Reply:
Done.
Thanks @ujjwalpl. I like the approach that you explained.